test/tools/analyze_tool_test.dart

```diff
@@ -163,19 +206,27 @@ void main() {
     test('with <=5 entrypoints', () {
       expect(Logger.root.onRecord,
           emitsThrough(infoLogOf(contains('dartanalyzer -t a b c d e'))));
-      logCommand(['-t'], ['a', 'b', 'c', 'd', 'e']);
+      logCommand(['-t'], ['a', 'b', 'c', 'd', 'e'], useDartAnalyzer: false);
     });
 
     test('with >5 entrypoints', () {
       expect(Logger.root.onRecord,
           emitsThrough(infoLogOf(contains('dartanalyzer -t <6 paths>'))));
-      logCommand(['-t'], ['a', 'b', 'c', 'd', 'e', 'f']);
+      logCommand(['-t'], ['a', 'b', 'c', 'd', 'e', 'f'],
+          useDartAnalyzer: false);
     });
 
     test('with >5 entrypoints in verbose mode', () {
       expect(Logger.root.onRecord,
           emitsThrough(infoLogOf(contains('dartanalyzer -t a b c d e f'))));
-      logCommand(['-t'], ['a', 'b', 'c', 'd', 'e', 'f'], verbose: true);
+      logCommand(['-t'], ['a', 'b', 'c', 'd', 'e', 'f'],
+          verbose: true, useDartAnalyzer: false);
+    });
+
+    test('in useDartAnalyze mode', () {
+      expect(
+          Logger.root.onRecord, emitsThrough(infoLogOf(contains('dart -t a'))));
```

Comment:
I think this should be:
```suggestion
          Logger.root.onRecord, emitsThrough(infoLogOf(contains('dart analyze -t a'))));
```